### ISSUE TITLE
option to download urls for feed items

### ISSUE THREAD

**Issue by gfa** [rss2email/rss2email#90]
Hello

this is a whistlist bug reported to Debian which I'm forwarding

>
> I use r2e to feed all RSS feeds into my mail reading routine, most
> of which happens while I am offline. Unfortunately, some important
> sites (most notably newspapers) only provide kicker paragraphs in
> the feed and require you to visit the URL to get the whole story.
> Even though I have tried to convince them to do otherwise, they are
> not interested (since they'd lose advertising revenue).S
> This means that for certain stories, I have to keep them around and
> visit them later when I am back online, and that can create quite
> a backlog.
> 
> If r2e would allow me to tag feeds such that it simply scrapes the
> URL for each feed and attach the entire HTML page to the individual
> mails it generates, the problem would be solved. Do you see this as
> a possible feature enhancement?
>

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=455977

**Comment by Profpatsch** [comment]
Looks like we should have a post-fetch hook (just a command that we call if it’s specified in the cofnig) which is passed feed item info as json … wait, isn’t there a thing for that already, where it allows you to modify your mail before it sends it?

**Comment by Profpatsch** [comment]
yes, we have: https://github.com/rss2email/rss2email#post-processing

**Comment by Ekleog** [comment]
This sounds relatively easy to do in a post-process hook, that we certainly could ship with rss2email if it's submitted as a PR :)
